openslides_backend/action/user/update_temporary.py

```diff
@@ -27,5 +27,7 @@ class UserUpdateTemporary(UpdateAction):
             "email",
             "vote_weight",
             "is_present_in_meeting_ids",
+            "default_password",
+            "group__ids",
```

Comment:
This is wrong I think. `group_$_ids` is a structured field and what you are doing here is writing directly into the field `group__ids` instead of using it as a template field. We should wait for the discussion in https://github.com/OpenSlides/openslides-backend/issues/219 until we change anything here.

Reply:
Update: discussion in https://github.com/OpenSlides-action-crew/openslides-backend-coordination/issues/135

Reply:
Okay, seems like we came to a conclusion there. They payload will have a field `group_ids` which you need to translate internally to `group_$<meeting_id>_ids` which will then be written to the DS.

Reply:
Ok.

Reply:
Ok, do it then, so I have an example how to handle this case.